.github/workflows/ubuntu-images.yaml

```diff
@@ -0,0 +1,61 @@
+name: "Images: Build and push Ubuntu toolbx images"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - images/ubuntu/**
+      - .github/workflows/ubuntu-images.yaml
+  schedule:
+    - cron: '0 0 * * MON'
+
+# Prevent multiple workflow runs from racing
+concurrency: ${{ github.workflow }}
+
+env:
+  latest_release: '22.04'
+
+jobs:
+  build-and-push-images:
+    strategy:
+      matrix:
+        release: ['16.04', '18.04', '20.04', '22.04', '22.10']
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: 'jmennius+toolbox_test'  # TODO: credentials
+          password: ${{ secrets.QUAY_TEST_ROBOT_TOKEN }}
```

Comment:
Shouldn't these be `toolbx-images+github` and `secrets.QUAY_ROBOT_TOKEN` ?

Of course, we might have to tweak the settings of this project for them to actually work, but it's time to get the code ready, no?

Reply:
Done, as @travier suggested.  It's actually `toolbx+github` now, not `toolbx-images+github`, because we want to publish these images at `quay.io/toolbx/ubuntu-toolbox`.